olm.sh

```diff
@@ -18,3 +18,21 @@ yq -i '.annotations."com.redhat.openshift.versions" |= "v4.8-v4.12"' bundles/red
 # Add needed annotations for redhat marketplace
 yq -i '.metadata.annotations."marketplace.openshift.io/remote-workflow" |= "https://marketplace.redhat.com/en-us/operators/minio-directpv-operator-rhmp/pricing?utm_source=openshift_console"' bundles/redhat-marketplace/"$RELEASE"/manifests/$package.clusterserviceversion.yaml
 yq -i '.metadata.annotations."marketplace.openshift.io/support-workflow" |= "https://marketplace.redhat.com/en-us/operators/minio-directpv-operator-rhmp/support?utm_source=openshift_console"' bundles/redhat-marketplace/"$RELEASE"/manifests/$package.clusterserviceversion.yaml
+
+# Use SHA Digest for CSI Provisioner Image
+csiProvisionerImage="quay.io/minio/csi-provisioner:v3.4.0"
+csiProvisionerImageDigest=$(docker pull "$csiProvisionerImage" | grep Digest | awk -F ' ' '{print $2}')
```

Comment:
1. We could use daemon-less `podman` instead of `docker`
2. The command may be simplified by `podman pull "$image" | awk '/Digest/ { print $2}'`